.github/workflows/one-click-hosting-release.yml

```diff
@@ -44,26 +44,30 @@ jobs:
           CHANNEL: ${{ env.CHANNEL }}
           PRERELEASE: ${{ github.event.inputs.pre_release }}
       - name: Env Vars Configure
+        id: env_vars_configure
         uses: ./.github/workflows/env-vars-configure
         with:
           CHANNEL: ${{ env.CHANNEL }}
           REF: ${{ github.ref }}
-          POSTFIX: ${{ env.POSTFIX }}
+          # POSTFIX: ${{ env.POSTFIX }}
+          POSTFIX: ${{ steps.env_vars_configure.outputs.POSTFIX }}
```

Comment:
This doesn't make sense to me - how can we use the output of the step we want to run?